Lombiq.NodeJs.Extensions/Docs/Styles.md

```diff
@@ -65,16 +65,20 @@ The `build:styles` script is a wrapper to execute the `lint:styles` and `compile
 The rules are found in 2 files:
 
 - _.stylelintrc.lombiq-base.js_: These rules are Lombiq overrides for [stylelint-config-standard-scss](https://www.npmjs.com/package/stylelint-config-standard-scss). You can find the file [here](../config/.stylelintrc.lombiq-base.js).
-- _.stylelintrc.js_: In this file you can define your own overriding rules.
+- _.stylelintrc.js_: In this file you can override the above Lombiq rules, or define your own [Stylelint configuration](https://stylelint.io/user-guide/configure/) altogether.
 
-The _.stylelintrc.js_ file initially extends _.stylelintrc.lombiq-base.js_ from the Node.js Extensions `npm` package. If you're using this project from a [submodule](../../Readme.md#as-a-git-submodule), that file will automatically be created in your project during the first build. Should you prefer to use a global _.stylelintrc.js_ file for your whole solution, you can instruct Node.js Extensions to create that file in the location specified by the MSBuild property `<NodeJsExtensionsGlobalStylelintConfigurationDirectory>`. This property is easiest added in a _Directory.Build.props_ file in your solution's root directory as follows:
+The _.stylelintrc.js_ file will automatically be created in your project during the first build. Please open it and adjust the path to _.stylelintrc.lombiq-base.js_ according to your solution's directory structure.
+
+### Using a solution-wide configuration
+
+> ℹ This option only works when using Node.js Extensions from a submodule, **not** from the NuGet package.
+
+In order to use a global _.stylelintrc.js_ file for your whole solution, you can instruct Node.js Extensions to create that file in the location specified by the MSBuild property `<NodeJsExtensionsGlobalStylelintConfigurationDirectory>`. This property is easiest to add in a _Directory.Build.props_ file in your solution's root directory as follows:
 
 ```xml
 <NodeJsExtensionsGlobalStylelintConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalStylelintConfigurationDirectory>
 ```
 
-> ℹ Please edit _.stylelintrc.js_ once it has been created, and adjust the path to _.stylelintrc.lombiq-base.js_ according to your solution's directory structure.
```

Comment:
Is this not needed anymore?

Reply:
I've moved this up a few lines: https://github.com/Lombiq/NodeJs-Extensions/pull/56/files/ffffe57f2ad93183dc20c156dd52a82ab6d5fd7d#diff-33acbba1e704c6e4f93652f05a1b55652325239733cf57e1c33da74bba369307R70.